File: app/src/main/java/xyz/untan/gyazauto/AppStatus.java
package xyz.untan.gyazauto;

import com.os.operando.garum.annotations.Pref;
import com.os.operando.garum.annotations.PrefKey;
import com.os.operando.garum.models.PrefModel;


@Pref(name = "app_status")
class AppStatus extends PrefModel{
    @PrefKey
    String accessToken;
}

File: app/src/main/java/xyz/untan/gyazauto/AppStatus.kt
package xyz.untan.gyazauto

import com.os.operando.garum.annotations.Pref
import com.os.operando.garum.annotations.PrefKey
import com.os.operando.garum.models.PrefModel

@Pref(name = "app_status")
internal class AppStatus : PrefModel() {
    @PrefKey
    var accessToken: String? = null
}
